Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -713,7 +723,7 @@ private static void AssertExpectedAnalysisSetting(BuildLog actualResult, string
 
         #region Setup
 
-        private BuildLog Execute_Settings_ValidSetup(string msBuildLanguage, bool isTestProject, string excludeTestProject)
+        private BuildLog Execute_Settings_ValidSetup(string msBuildLanguage, bool isTestProject, string excludeTestProject, string csRuleSetPath = @"d:\C#-normal.ruleset", string vbRulesetPath = @"d:\VB-normal.ruleset")
```

Comment:
this is invoked only once w/o params - inside `Settings_ValidSetup_ForExcludedTestProject`

so it would make sense to avoid default params and move those values inside the caller

also, it would be more readable to have the ruleset path `"foo-cs.ruleset" and `foo-vb.ruleset` to underline they're dummy paths.

otherwise, one might thing that the tests need a `d:\` drive to run them